garage/replay_buffer/simple_replay_buffer.py

```diff
@@ -25,16 +21,16 @@ def sample(self, batch_size):
             buffer[key] = self._buffer[key][:self._current_size]
 
         # Select which episodes to use
-        time_horizon = buffer["action"].shape[1]
-        rollout_batch_size = buffer["action"].shape[0]
+        time_horizon = buffer['action'].shape[1]
```

Comment:
Is it still valid to have `time_horizon` since the replay buffer now stores variable-length episodes? It may beyond the scope of this PR. I just think the interface seems a bit confusing now.

Reply:
Is it because there is no `max_path_length` anymore?